packages/packagers/json/src/JSONPackager.js

```diff
@@ -0,0 +1,24 @@
+// @flow strict-local
+
+import assert from 'assert';
+import {Packager} from '@parcel/plugin';
+import {replaceURLReferences} from '@parcel/utils';
+
+export default (new Packager({
```

Comment:
As I've written in the default config, this is probably not needed.

Reply:
[Rationale](https://github.com/parcel-bundler/parcel/pull/5304#discussion_r523477732)